app/change_set_persisters/change_set_persister/geoserver_publish.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+class ChangeSetPersister
+  class GeoserverPublish
+    class Factory
+      attr_reader :operation
+      def initialize(operation:)
+        @operation = operation
+      end
+
+      def new(*args)
+        klass.new(*args)
+      end
+
+      private
+
+        def klass_name
+          "ChangeSetPersister::GeoserverPublish#{operation.to_s.camelize}"
+        end
+
+        def klass
+          klass_name.constantize
+        rescue
+          raise NotImplementedError, "#{klass_name} not supported as a change set persistence handler"
+        end
+    end
+  end
+
+  class GeoserverPublishUpdate
+    attr_reader :change_set_persister, :change_set, :post_save_resource
+    def initialize(change_set_persister:, change_set:, post_save_resource: nil)
+      @change_set = change_set
+      @change_set_persister = change_set_persister
```

Comment:
Do we need this since it's not used? and the attr_reader entry also

Reply:
Removed.